packages/cashier/src/types/stores/root-store.types.ts

```diff
@@ -0,0 +1,10 @@
+import { TClientStore } from './client-store.types';
+import { TCommonStore } from './common-store.types';
+import { TUiStore } from './ui-store.types';
+
+export type TRootStore = {
```

Comment:
@george-usynin-binary We need to also rename it everywhere we have imported `RootStore` from `Types`

Reply:
@farzin-fs I think this will be applied in the TS migration PR of each part. It's better not to change them in this PR (if you mean this) due to conflicts.

Reply:
@hamid-deriv Yeah we can also do that 🤔
Just we need to make sure we won’t miss any files since we have added new TS errors in this PR by renaming the type.

Reply:
Until now, we don't use TRootStore in any file on master, so after this PR merged, we need to fix this in any new PR. :+1: